content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Quality Gates
+kind: documentation
+description: Learn how to use Quality Gates in your pipeline.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/tests"
+  tag: "Documentation"
+  text: "Learn about Test Visibility"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="#" header="false" >}}
+Quality Gates are in beta.
+{{< /callout >}}
+
+Quality Gates allows you to gate your workflows based on signals in Datadog.
+For example, you can block code from being merged if it introduces new [flaky tests][8], even if the tests are passing.
+
+Using Quality Gates, you have control over what is merged into the default branch, and ultimately on what is deployed.
+This allows you to ensure that the code running in production is meeting high quality standards, reducing incidents and unwanted behaviors.
+
+## Set up Quality Gates
+
+There are two main steps required to set up Quality Gates:
+1. Create one or more rules.
+2. Add the `datadog-ci gate evaluate` command in your CI pipeline.
+
+## Create a rule
+
+To create Quality Gates rules for your organization, your user account must have the `quality_gate_rules_write` [permission][1].
+
+1. In Datadog, navigate to [**CI** > **Quality Gate Rules**][2] and click **+ New Rule**.
```

Comment:
we can add a link for "New Rule" as well, wdyt?

Reply:
Sounds good, do you have it handy?

Reply:
I can add it there, but I don't know if it will help much. Also, I'm not sure what it will show if customers won't have the permission to create rules, whereas the main rules page handles that well.

Reply:
ok, I agree that it might be confusing for users without permissions